arch/arm/src/samv7/sam_pwm.c

```diff
@@ -957,79 +957,79 @@ static int pwm_start(struct pwm_lowerhalf_s *dev,
+          pwm_set_polarity(dev, priv->channels[index - 1].channel,
+                            info->channels[i].cpol,
+                            info->channels[i].dcpol);
+          pwm_set_output(dev, priv->channels[index - 1].channel,
+                          info->channels[i].duty);
 #ifdef CONFIG_PWM_OVERWRITE
-              if (info->channels[i].ch_outp_ovrwr)
-                {
-                  regval = pwm_getreg(priv, SAMV7_PWM_OOV);
-                  regval &= ~(info->channels[i].ch_outp_ovrwr_val
-                      << priv->channels[i].channel);
-                  pwm_putreg(priv, SAMV7_PWM_OOV, regval);
-
-                  regval = (1 << priv->channels[i].channel);
-                  pwm_putreg(priv, SAMV7_PWM_OSS, regval);
-                }
-              else
-                {
-                  /* Release overwrite of channel */
-
-                  regval = (1 << priv->channels[i].channel);
-                  pwm_putreg(priv, SAMV7_PWM_OSC, regval);
-                }
-#endif
+          if (info->channels[i].ch_outp_ovrwr)
+            {
+              regval = pwm_getreg(priv, SAMV7_PWM_OOV);
+              regval &= ~(info->channels[i].ch_outp_ovrwr_val
+                  << priv->channels[i].channel);
+              pwm_putreg(priv, SAMV7_PWM_OOV, regval);
+
+              regval = (1 << priv->channels[i].channel);
+              pwm_putreg(priv, SAMV7_PWM_OSS, regval);
+            }
+          else
+            {
+              /* Release overwrite of channel */
+
+              regval = (1 << priv->channels[i].channel);
+              pwm_putreg(priv, SAMV7_PWM_OSC, regval);
             }
+#endif
         }
+    }
 
-      /* Perform the update of synchronized PWM channels */
+  /* Perform the update of synchronized PWM channels */
 
-      if (priv->sync)
-        {
-          regval = SCUC_UPDULOCK;
-
-          /* Enable the Channel 0 if synchronous channels are used.
-           * Channel 0's counter is used by all synchronous channels and
-           * enabling CH0 results in enabling all synchronous channels.
-           *
-           * Enable the CH0 here after all setting all channel parameters,
-           * because setting polarity configurations requires disabled
-           * channels.
-           */
-
-          pwm_putreg(priv, SAMV7_PWM_ENA, CHID_SEL(1));
-          pwm_putreg(priv, SAMV7_PWM_SCUC, regval);
-        }
+  if (priv->sync)
+    {
+      regval = SCUC_UPDULOCK;
+
+      /* Enable the Channel 0 if synchronous channels are used.
+       * Channel 0's counter is used by all synchronous channels and
+       * enabling CH0 results in enabling all synchronous channels.
+       *
+       * Enable the CH0 here after all setting all channel parameters,
+       * because setting polarity configurations requires disabled
+       * channels.
+       */
+
+      pwm_putreg(priv, SAMV7_PWM_ENA, CHID_SEL(1));
+      pwm_putreg(priv, SAMV7_PWM_SCUC, regval);
+    }
```

Comment:
@michallenc could you do me a favour and fix style in the `#else` case as well?

Reply:
Done.